pkg/operator/controllers/alertwebhook/alertwebhook_controller.go

```diff
@@ -26,23 +26,24 @@ import (
 
 var alertManagerName = types.NamespacedName{Name: "alertmanager-main", Namespace: "openshift-monitoring"}
 
-// AlertWebhookReconciler reconciles the alertmanager webhook
-type AlertWebhookReconciler struct {
+// Reconciler reconciles the alertmanager webhook
+type Reconciler struct {
+	log *logrus.Entry
+
```

Comment:
Should we potentially remove the whitespaces here for legibility? I see the spaces are consistent across all struct types, but the same spaces are not present in the return &Reconciler statements.

Reply:
There is a (weak) pattern in this repo of putting fields like `log` and `env` in the first group, and then grouping other related fields together as well (based on categorization like, "these are all clientsets").

The newline makes it easier for me, and apparently others, to read, but I suppose it's just a matter of preference. The most legible thing would be to have a standard of some kind, whether it's this one or a different one.

The rationale behind omitting the spaces in the returned struct is that it is already spaced for readability in the definition.